vscode/src/extension.ts

```diff
@@ -609,7 +609,9 @@ class VsCodeExtension {
       const credentials = await checkAndPromptForCredentials(this.context, this.state.logger);
       if (!credentials) {
         this.state.mutateData((draft) => {
-          draft.configErrors.push(createConfigError.missingAuthCredentials());
+          if (!draft.configErrors.some((error) => error.type === "missing-auth-credentials")) {
```

Comment:
I think this is good for now but filing an issue to make configErrors a managed set so we don't always to have make these checks